### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by fhunleth** [nerves-project/nerves_network#15]
Add support for IPv6. This should be done in a way where IPv4 can still be used. Given that IPv6 could include a lot of things, for this issue, I'm just looking for a way to to get IPv6 addresses assigned to interfaces whether they be static ones, link local or DHCP-based.

**Comment by jamesotron** [comment]
IPv6 autoconfiguration is also pretty darn useful. It should automatically detect RA's and configure itself accordingly.

**Comment by fhunleth** [comment]
Yes, definitely! I know of two companies that use IPv6 w/ Nerves internally. Neither has opensourced their code yet, but I'm really hopefully to get at least one to do it. I'll link to it here when that happens or if anyone beats them to it.

**Comment by ConnorRigby** [comment]
@michaelkschmidt has brought it to my attention that [a fork he works with](https://github.com/se-apc/nerves_network) has support for ipv6. Will circle back to this.
